Qfront/src/redux/orders/Orders.js

```diff
@@ -0,0 +1,47 @@
+import { createAuthOrder, createGuestOrder } from "./actions";
+import { createSlice } from "@reduxjs/toolkit";
+
+const initialState = {
+    success: null,
+    loading: false,
+    authData : [],
+    guestData : [],
+    authDataError : null,
+    guestDataError : null,
+};
+
+const orderSlice = createSlice({
+    name: "order",
+    initialState,
+    reducers: {},
+    extraReducers: (builder) => {
+        builder.addCase(createAuthOrder.fulfilled, (state, action) => {
```

Comment:
In fulfilled, we should set error state to their default state. Please make this change in all other reducers necessary